Views/Shared/_Layout.cshtml

```diff
@@ -23,14 +23,17 @@
                             <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Index">Home</a>
                         </li>
                         <li class="nav-item">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
+                            @if (User.IsInRole("Teacher"))
+                            {
+                                <a class="nav-link text-dark" asp-area="" asp-controller="SystemUser" asp-action="Index">Users</a>
+                            }
                         </li>
                         <li class="nav-item">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="SystemUser" asp-action="Index">Users</a>
-                        </li>
```

Comment:
We still need to show teachers the list of users.

Reply:
My mistake, I misinterpreted the way the changes were presented. This is already done.